### PR TITLE
chore(ios): Changes required for XCode 14.3

### PR DESCRIPTION
**Pull request by davidmoore1**
Update Carthage dependencies to versions that will build under XCode 14.3

# User Testing

(adopted from Keyman for iPhone and iPad Acceptance Test Procedures)

---

## Gather Assets for Testing ##
- [ ] We only need to test all these on iPhone 
- [ ] Record device's iOS version
- [ ] Ensure the device is properly configured for testing and is registered with TestFlight. 

At this time, Keyman for iOS does not support external keyboard input.

## Setup Steps ##
1.  Uninstall previous version of Keyman on the device (on a simulator, "Erase All Content and Settings..." under Device menu)
2. Install test build IPA (on a simulator, "iOS Simulator Image")
3. Disable internet connectivity of the device
4. Start Keyman and select **Add a keyboard for your language**
5. Verify error message "Could not reach Keyman server!" or "The Internet connection appears to be offline." is displayed
6. Click OK to dismiss.  Close Keyman app
7. On device, enable internet connectivity
8. Start Keyman
9. On initial **Get Started** menu, select "Enable Keyman as system-wide keyboard" and do any necessary system steps to enable Keyman

---

### Test Cases
- **TEST_GET_STARTED**: Get Started
  - Verify touching **Add a keyboard for your language** pulls up the **keyboard search**
      - This should also be accessible as **Settings** > **Installed Languages** > **`+`**.
  - Verify touching **Enable Keyman as system-wide keyboard** pulls up Keyman's menu within iOS settings.  (11.0+ devices)
      - If older versions of iOS are supported, ensure that the help page for doing it manually is properly linked.
  - Verify touching **More info** brings up an accurate version and the main table-of-contents help page.
      - This should also be accessible as **Info** from the main menu dropdown (on phones) or as the circled-i button (on tablets).
  - Set **Don't show again** to checked, close and re-open Keyman, and verify **Get Started** menu does not appear

- **TEST_ADD_NEW_KEYBOARD**: Add New Keyboard
This menu is accessed via **Get Started** menu or **Settings** menu
  - On the device, enable internet connectivity
  - Verify a new keyboard can be downloaded and selected, say "US Basic"
  - Return to the text input screen and confirm that it operates correctly.
  - Without changing selected language, confirm that it can be cleanly deleted via the Settings menu.
  - Confirm that another keyboard is correctly selected afterward.

- **TEST_ADD_NEW_KEYBOARD_FROM_FILE**:
  - On the device, go to keyman.com on Safari
  - Search for "US Basic"
  - scroll down to keyboard details and tap to download the keyboard package (i.e. `.kmp` file)
  - open Files app and navigate to Download folder to find "basic_kbdus.kmp" and tap on it to start the installation processes
  - Keyman app should be pulled up
  - follow the screen to finish the keyboard installation
  - verity that the keyboard is correctly installed and working as expected

- **TEST_SWITCH_KEYBOARD**
  - press on the globe key
  - the keyboard menu should be shown
  - press on a keyboard to switch to, say "EuroLatin (SIL)"
  - confirm that "EuroLatin (SIL)" is now selected and ready to be used
  - swich back to "US Basic" and confirm it does as expected

- **TEST_SHARE**: Share
  - Type and verify text can be copied and shared to external app

- **TEST_KEYMAN_BROWSER**: Keyman Browser ###
  - On default page, click on text field and set system keyboard to Keyman
  - Verify can type with Keyman as system keyboard
  - Verify that predictive suggestions are properly applied when selected (using `sil_euro_latin` + English)
  - Close and reopen Keyman app

- **TEST_TEXT_SIZE**: Text Size
  - Verify text can be rescaled from Text Size 9 to 72

- **TEST_CLEAR_TEXT**: Clear Text
  - Verify text can be cleared

- **TEST_INFO**: Info
  - Verify that the Keyman iOS version is displayed; record the version
  - Verify that the Info menu options matches that of the **More Info...** option from the **Get Started** menu option.
  - Verify that the Keyman iOS version matches the version string seen at (Home Screen) > [iOS] Settings > Keyman.

- **TEST_SETTINGS**: Settings
  - Installed Languages ... Add Language
  - Select an installed language ... Add (another) Keyboard
    - Should automatically perform a filtered search based upon that language's BCP-47 code, i.e. for "English" it should be seen as shown in the screenshot below.
    

  - Select an installed language ... Dictionary
    - Turn on/off 'Enable Predictions' and validate each
    - Turn on/off 'Enable Corrections' and validate each
  - Validate that when both predictions and corrections are off, banner is not visible
  - Add/Remove dictionaries - validate
  - If multiple dictionaries are available, test swapping between them

---

### Test Cases

- **TEST_US_BASIC**: English: `US Basic` in-app
  - Verify that the **US Basic** keyboard can be downloaded through Settings > Installed Languages > English > `+`
    - This should automatically bring up search results for **English** keyboards.
    - When the package-installer launches, English should be the one pre-selected language.
  - Install the package for both English and Tagalog by selecting both of them in the installation processes.
  - Set **English: US Basic** as the active in-app keyboard.
  - Verify that predictive text suggestions appear.
  - Set **Tagalog: US Basic** as the active in-app keyboard.
  - Verify that predictive text suggestions do _not_ appear.
  - Verify that long-press `123` key works, presenting options for a currency layer and a symbol layer.
  - Select the currency layer, then press the `£` key.
  - Verify that the keyboard outputs correctly and returns to the default layer.
  - Go to the symbol layer and press the `©` key.
  - Verify that the keyboard outputs correctly and remains on the symbol layer.
  - Revert to the default layer and ensure basic key inputs work.
  - Long-press `e` on the default layer and select a subkey.
  - Verify that the selected key produces the correct output.
  - From the `Settings > Languages > English > US Basic` menu, follow the "Help link" and ensure it displays appropriate help.
    - (Should work both online and offline)
  - From the `Settings > Languages > English > US Basic` menu, scan the QR code with a phone and test that it links to the current version of that keyboard's public download page on keyman.com.
  - Returning to the app, uninstall the **US Basic** keyboard via Settings for Tagalog.
  - Set **English: US Basic** as the active in-app keyboard.
  - Verify that the keyboard appears correctly and ensure basic key inputs work.
    - This is to ensure that uninstalling for one language doesn't adversely affect installations of the same resource for another language.
  - Uninstall the **US Basic** keyboard via Settings for English.

- **TEST_EUROLATIN**: `English (Eurolatin)` in-app
  - In portrait orientation, verify that the OSK appears and fills the width the bottom of the screen
  - Verify long-press `q` key works
  - Verify long-press `k` key works
      - Verify that any predictive suggestions generated feel 'reasonable' (based upon 'k', the selected subkey, or one of its neighbors)
  - Verify uppercase layer can be selected via `SHIFT`
  - Verify number layer can be selected via `123`
  - Verify long-press `1` key works
  - Verify long-press `0` key works
  - Verify backspace, space, and enter keys work

---

## SUITE_SYSTEM_KEYBOARD:

Test the following after setting Keyman as a system keyboard.

Test Cases
- **TEST_BASIC_USAGE**: ensure that the keyboards functions as expected on all layers. Please test on multiple keyboards, i.e. EuroLatin (SIL), Cameroon QWERTY keyboard and Khmer Angkor.

- **TEST_LONG_PRESS**: ensure that longpress on all applicable layers are working as intended.

- **TEST_GLOBE_KEY**: ensure that a short press on the globe key switch to the next keyboard and a longpress on it open the keyboard menu where you can select a keyboard of your choice

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: ios/Cartfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 github "marmelroy/Zip"
-github "DaveWoodCom/XCGLogger" ~> 6.1.0
+github "keymanapp/dependency-XCGLogger" "master"
```
Comment (darcywong00) [inline ios/Cartfile:2]: Do we want a specific tag here? The latest looks to be 7.0.1 from Feb 2020

**Comment by darcywong00** [comment]
> Update Carthage dependencies to versions that will build under XCode 14.3

I see this includes updating sentry-cocoa to 8.7.0.

File: ios/Cartfile (hunk 1 above)
Comment (mcdurdin) [reply ios/Cartfile:2]: I think we can stick with 'master' given we control the fork

Comment (mcdurdin) [reply ios/Cartfile:2]: You use 'master' here but 'head' in oem/firstvoices/ios/Cartfile?

```
github "keymanapp/dependency-XCGLogger" "master"
```

vs 

```
github "keymanapp/dependency-XCGLogger" "head"
```

Can we use the same in both places please?

**Review by mcdurdin — COMMENTED** [review]
I'll kick off a build

**Comment by mcdurdin** [comment]
Thanks @davidmoore1 for your PR -- really appreciate your contribution!

**Review by mcdurdin — CHANGES_REQUESTED** [review]
Just that one change requested above

**Comment by MakaraSok** [comment]
(adopted from Keyman for iPhone and iPad Acceptance Test Procedures)

---

# User Testing
## Gather Assets for Testing ##
- [ ] Record device's iOS version
- [ ] Ensure the device is properly configured for testing and is registered with TestFlight.

At this time, Keyman for iOS does not support external keyboard input.

## Setup Steps ##
1.  Uninstall previous version of Keyman on the device (on a simulator, "Erase All Content and Settings..." under Device menu)
2. Install test build IPA (on a simulator, "iOS Simulator Image")
3. Disable internet connectivity of the device
4. Start Keyman and select **Add a keyboard for your language**
5. Verify error message "Could not reach Keyman server!" or "The Internet connection appears to be offline." is displayed
6. Click OK to dismiss.  Close Keyman app
7. On device, enable internet connectivity
8. Start Keyman
9. On initial **Get Started** menu, select "Enable Keyman as system-wide keyboard" and do any necessary system steps to enable Keyman

---

## SUITE_MENU_FUNCTIONALITY: Menu Functionality Tests, in-app
- GROUP_IPHONE_PORTRAIT: portrait mode in iPhone
- GROUP_IPHONE_LANDSCAPE: landscape mode in iPhone
- GROUP_IPAD_PORTRAIT: portrait mode in iPad
- GROUP_IPAD_LANDSCAPE: landscape mode in iPad

### Test Cases
- **TEST_GET_STARTED**: Get Started
  - Verify touching **Add a keyboard for your language** pulls up the **keyboard search**
      - This should also be accessible as **Settings** > **Installed Languages** > **`+`**.
  - Verify touching **Enable Keyman as system-wide keyboard** pulls up Keyman's menu within iOS settings.  (11.0+ devices)
      - If older versions of iOS are supported, ensure that the help page for doing it manually is properly linked.
  - Verify touching **More info** brings up an accurate version and the main table-of-contents help page.
      - This should also be accessible as **Info** from the main menu dropdown (on phones) or as the circled-i button (on tablets).
  - Set **Don't show again** to checked, close and re-open Keyman, and verify **Get Started** menu does not appear

- **TEST_ADD_NEW_KEYBOARD**: Add New Keyboard
This menu is accessed via **Get Started** menu or **Settings** menu
  - On the device, enable internet connectivity
  - Verify a new keyboard can be downloaded and selected, say "US Basic"
  - Return to the text input screen and confirm that it operates correctly.
  - Without changing selected language, confirm that it can be cleanly deleted via the Settings menu.
  - Confirm that another keyboard is correctly selected afterward.

- **TEST_ADD_NEW_KEYBOARD_FROM_FILE**:
  - On the device, go to keyman.com on Safari
  - Search for "US Basic"
  - scroll down to keyboard details and tap to download the keyboard package (i.e. `.kmp` file)
  - open Files app and navigate to Download folder to find "basic_kbdus.kmp" and tap on it to start the installation processes
  - Keyman app should be pulled up
  - follow the screen to finish the keyboard installation
  - verity that the keyboard is correctly installed and working as expected

- **TEST_SWITCH_KEYBOARD**
  - press on the globe key
  - the keyboard menu should be shown
  - press on a keyboard to switch to, say "EuroLatin (SIL)"
  - confirm that "EuroLatin (SIL)" is now selected and ready to be used
  - swich back to "US Basic" and confirm it does as expected

- **TEST_SHARE**: Share
  - Type and verify text can be copied and shared to external app

- **TEST_KEYMAN_BROWSER**: Keyman Browser ###
  - On default page, click on text field and set system keyboard to Keyman
  - Verify can type with Keyman as system keyboard
  - Verify that predictive suggestions are properly applied when selected (using `sil_euro_latin` + English)
  - Close and reopen Keyman app

- **TEST_TEXT_SIZE**: Text Size
  - Verify text can be rescaled from Text Size 9 to 72

- **TEST_CLEAR_TEXT**: Clear Text
  - Verify text can be cleared

- **TEST_INFO**: Info
  - Verify that the Keyman iOS version is displayed; record the version
  - Verify that the Info menu options matches that of the **More Info...** option from the **Get Started** menu option.
  - Verify that the Keyman iOS version matches the version string seen at (Home Screen) > [iOS] Settings > Keyman.

- **TEST_SETTINGS**: Settings
  - Installed Languages ... Add Language
  - Select an installed language ... Add (another) Keyboard
    - Should automatically perform a filtered search based upon that language's BCP-47 code, i.e. for "English" it should be seen as shown in the screenshot below.
    

  - Select an installed language ... Dictionary
    - Turn on/off 'Enable Predictions' and validate each
    - Turn on/off 'Enable Corrections' and validate each
  - Validate that when both predictions and corrections are off, banner is not visible
  - Add/Remove dictionaries - validate
  - If multiple dictionaries are available, test swapping between them

---

## SUITE_IN_APP_PHONE_ONLY: 
- GROUP_IPHONE_PORTRAIT: portrait mode in iPhone
- GROUP_IPHONE_LANDSCAPE: landscape mode in iPhone

### Test Cases

- **TEST_US_BASIC**: English: `US Basic` in-app
  - Verify that the **US Basic** keyboard can be downloaded through Settings > Installed Languages > English > `+`
    - This should automatically bring up search results for **English** keyboards.
    - When the package-installer launches, English should be the one pre-selected language.
  - Install the package for both English and Tagalog by selecting both of them in the installation processes.
  - Set **English: US Basic** as the active in-app keyboard.
  - Verify that predictive text suggestions appear.
  - Set **Tagalog: US Basic** as the active in-app keyboard.
  - Verify that predictive text suggestions do _not_ appear.
  - Verify that long-press `123` key works, presenting options for a currency layer and a symbol layer.
  - Select the currency layer, then press the `£` key.
  - Verify that the keyboard outputs correctly and returns to the default layer.
  - Go to the symbol layer and press the `©` key.
  - Verify that the keyboard outputs correctly and remains on the symbol layer.
  - Revert to the default layer and ensure basic key inputs work.
  - Long-press `e` on the default layer and select a subkey.
  - Verify that the selected key produces the correct output.
  - From the `Settings > Languages > English > US Basic` menu, follow the "Help link" and ensure it displays appropriate help.
    - (Should work both online and offline)
  - From the `Settings > Languages > English > US Basic` menu, scan the QR code with a phone and test that it links to the current version of that keyboard's public download page on keyman.com.
  - Returning to the app, uninstall the **US Basic** keyboard via Settings for Tagalog.
  - Set **English: US Basic** as the active in-app keyboard.
  - Verify that the keyboard appears correctly and ensure basic key inputs work.
    - This is to ensure that uninstalling for one language doesn't adversely affect installations of the same resource for another language.
  - Uninstall the **US Basic** keyboard via Settings for English.

---

## SUITE_IN_APP_KEYBOARD:
- GROUP_IPHONE_PORTRAIT: portrait mode in iPhone
- GROUP_IPHONE_LANDSCAPE: landscape mode in iPhone
- GROUP_IPAD_PORTRAIT: portrait mode in iPad
- GROUP_IPAD_LANDSCAPE: landscape mode in iPad

### Test Cases

- **TEST_EUROLATIN**: `English (Eurolatin)` in-app
  - In portrait orientation, verify that the OSK appears and fills the width the bottom of the screen
  - Verify long-press `q` key works
  - Verify long-press `k` key works
      - Verify that any predictive suggestions generated feel 'reasonable' (based upon 'k', the selected subkey, or one of its neighbors)
  - Verify uppercase layer can be selected via `SHIFT`
  - Verify number layer can be selected via `123`
  - Verify long-press `1` key works
  - Verify long-press `0` key works
  - Verify backspace, space, and enter keys work

---

## SUITE_SYSTEM_KEYBOARD:
- GROUP_IPHONE_PORTRAIT: portrait mode in iPhone
- GROUP_IPHONE_LANDSCAPE: landscape mode in iPhone
- GROUP_IPAD_PORTRAIT: portrait mode in iPad
- GROUP_IPAD_LANDSCAPE: landscape mode in iPad

Test the following after setting Keyman as a system keyboard.

Test Cases
- **TEST_BASIC_USAGE**: ensure that the keyboards functions as expected on all layers. Please test on multiple keyboards, i.e. EuroLatin (SIL), Cameroon QWERTY keyboard and Khmer Angkor.
- **TEST_LONG_PRESS**: ensure that longpress on all applicable layers are working as intended.
    - Note that `khmer_angkor` lacks longpress keys on tablet form-factor devices, though the other two listed above still have them.
- **TEST_GLOBE_KEY**: ensure that a short press on the globe key switch to the next keyboard and a longpress on it open the keyboard menu where you can select a keyboard of your choice

## SUITE_SWITCH_LOCALE
- GROUP_IPHONE_PORTRAIT: portrait mode in iPhone
- GROUP_IPHONE_LANDSCAPE: landscape mode in iPhone
- GROUP_IPAD_PORTRAIT: portrait mode in iPad
- GROUP_IPAD_LANDSCAPE: landscape mode in iPad

### Test Cases
- **TEST_KM_LOCALE** 
  - from System Preferences, switch the UI language to Khmer by adding it in `Language & Region` using `+`
  - drag Khmer to the top of the list to set it as a primary UI language
  - check the dropdown menu to see it in Khmer 
  - verify the UI language switch to Khmer
- **TEST_AM_LOCALE**
  - from System Preferences, switch the UI language to Amharic by adding it in `Language & Region` using `+`
  - drag Amharic to the top of the list to set it as a primary UI language
  - check the dropdown menu to see it in Amharic
  - verify the UI language switch to Amharic

---

## SUITE_PREDICTIVE_TEXT_AND_AUTO_CORRECTION: Predictive Text and Auto Correction

- GROUP_IPHONE_PORTRAIT: portrait mode in iPhone
- GROUP_IPHONE_LANDSCAPE: landscape mode in iPhone
- GROUP_IPAD_PORTRAIT: portrait mode in iPad
- GROUP_IPAD_LANDSCAPE: landscape mode in iPad

### Test Cases

- **TEST_BROKEN_SUGGESTION**:  Verify that the following reproduction is no longer affected by this [bug](https://github.com/keymanapp/keyman/issues/7167).

  1. In Keyman for iOS, type <kbd>P</kbd><kbd>r</kbd><kbd>o</kbd><kbd>b</kbd> using the default keyboard (EuroLatin).
  2. Select `Problem` from banner.
  3. Press <kbd>Backspace</kbd>, then select `"Prob"` (undoing the last suggestion).
  4. Then press <kbd>Spacebar</kbd>.
  5. You should see  `Prob `, as if no suggestion were auto-applied from the spacebar.

- **TEST_QUOTE_O**:  Using the default English keyboard for your test environment, type `'` followed immediately by `o`.
  - If most of the displayed suggestions do not begin with an 'o', _**FAIL**_ this test.
    - Do not fail if "I" is one of them - it's a super-common word, and 'o' does neighbor 'i'.
  - Apply any of the displayed suggestions.  If the `'` mark is replaced, _**FAIL*__ this test.
  - _(nts: This test and the other two below are adopted from https://github.com/keymanapp/keyman/pull/7205.)_

- **TEST_QUOTE_OP**:  Using the default English keyboard for your test environment, type `'` followed immediately by `o`, then `p`.
  - If most of the displayed suggestions do not begin with an 'op', _**FAIL**_ this test.
    - Do not fail if "OP" is suggested, despite its capitalization.
  - Apply any of the displayed suggestions.  If the `'` mark is replaced, _**FAIL**_ this test.

- **TEST_OP**:  Using the default English keyboard for your test environment, type `you`, add a space, then type `o`, then `p`.
  - If most of the displayed suggestions do not begin with an `op`, _**FAIL**_ this test.
    - Do not fail if "OP" is suggested, despite its capitalization.
  - Apply any of the displayed suggestions.  If unwanted effects occur, _**FAIL**_ this test.
    - No prior whitespace should be deleted and the suggestion should be cleanly applied.
    - Example:  applying `open` to `you op` should result in `you open ` for the full context.

- **TEST_NW_Q**:  Using the default (`sil_euro_latin`) keyboard, press the far upper-left of the 'q' (or 'Q') key.
  - All suggestions should begin with the corresponding letter and case.  (So, 'question' for 'q' or 'Question' for 'Q'.)
  - _(nts: This test and the one below are adopted from https://github.com/keymanapp/keyman/pull/7241.)_

- **TEST_E_P**:  Using the default (`sil_euro_latin`) keyboard, press the far upper-right of the 'p' (or 'P') key.
  - All suggestions should begin with the corresponding letter and case.

- **TEST_GUILLEMET**: Guillemets («») are used to quote a speech in some language, i.e. Khmer and Jarai. We would like to ensure that the quillemet won't get delete when a predictive text is in action.
  - In Keyman for iOS app with [Jarai](https://keyman.com/keyboards/sil_jarai?bcp47=jra-khmr) keyboard active, go to the shift layer and press `«` on the first row.
  - Now press `ស` (the second key from the left on the third row) and pick the first option on the suggestion banner.
  - FAIL this test if the guillemet disappears.

**Comment by MakaraSok** [comment]
A full set of tests for iOS is included here to avoid any mishaps.

**Comment by bharanidharanj** [comment]
## SUITE_MENU_FUNCTIONALITY: Menu Functionality Tests, in-app

* **TEST_GET_STARTED (PASSED):** Tested with the attached PR build (Keyman 17.0.103-alpha-test-8746) in iOS 15.0 / iPhone 12 Pro Max Simulator and here is my observation: 1. Verified that the options ‘Add a keyboard for your languages’, ‘Setup Keyman as system-wide keyboard’ and ‘More Info’ are working as expected. Also, ‘Don’t show again’ option is also working as expected. 
* **TEST_ADD_NEW_KEYBOARD (PASSED):** Verified that the US Basic Keyboard appears as the active keyboard in the text input screen.
* **TEST_ADD_NEW_KEYBOARD_FROM_FILE (PASSED):** Verified that the US Basic Keyboard (from .kmp file)  installed successfully. Also, verified that the US Basic keyboard appears as an active keyboard.
* **TEST_SWITCH_KEYBOARD (PASSED):** Verified that the Switching keyboards using globe key is working as expected.
* **TEST_SHARE (PASSED):** Verified that the copied text “Keyman” appears on the Messages / Text box. Seems to be working as expected.
* **TEST_KEYMAN_BROWSER (PASSED):** Verified that the Predictive text (“Key Keys Keyboard) appears on the suggestion banner of the On Screen Keyboard. Seems to be working as expected.
* **TEST_TEXT_SIZE (PASSED):** Able to change the text size from small size to large size in the text input screen. Seems to be working as expected.
* **TEST_CLEAR_TEXT (PASSED):** Verified that the typed text message has been cleared from the Input text screen.
* **TEST_INFO (PASSED):** Verified that the Info Help topic is working fine in the Get Started dialog as well as in the Keyman menu list. Seems to be working as expected.
  
 




* **TEST_SETTINGS (PASSED):** Verified that the Settings / Installed Languages menu is working as expected.

**Comment by bharanidharanj** [comment]
# Test Results

## SUITE_IN_APP_KEYBOARD: 

* **TEST_US_BASIC (PASSED):** Tested with the attached PR build (Keyman 17.0.103-alpha-test-8746) in iOS 15.0 / iPhone 12 Pro Max Simulator and here is my observation: 1. Verified that the US Basic - English Keyboard and US Basic - Tagalog keyboard are working as expected.
* **TEST_EUROLATIN (PASSED):** Verified that the Short Press key, Long Press key, Number key, Backspace key, Spacebar key are working as expected in the Eurolatin (SIL) keyboard.

## SUITE_SYSTEM_KEYBOARD: 

* **TEST_BASIC_USAGE (PASSED):** Tested with the attached PR build (Keyman 17.0.103-alpha-test-8746) in iOS 15.0 / iPhone 12 Pro Max Simulator and here is my observation: 1. Verified that keyboards like Khmer Angkor, Cameroon QWERTY keyboards are working in all layers. Long Press, Short Press, Num Keys are also working fine with these keyboards. Seems to be working as expected.
* **TEST_LONG_PRESS (PASSED):** Verified that Long Press functionality is working fine in all layers. Seems to be working as expected.
* **TEST_GLOBE_KEY (PASSED):** Short Pressing the globe key switches to the next keyboard. Long Pressing the globe key shows the picker menu on the OSK, after setting Keyman as the System wide keyboard. Seems to be working as expected.

File: oem/firstvoices/ios/Cartfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-github "DaveWoodCom/XCGLogger" ~> 6.1.0
+github "keymanapp/dependency-XCGLogger" "head"
```
Comment (mcdurdin) [inline oem/firstvoices/ios/Cartfile:1]:
```suggestion
github "keymanapp/dependency-XCGLogger" "master"
```

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.108-alpha](https://keyman.com/downloads/releases/alpha/17.0.108)

**Pull request merged**